### DISCUSSION TITLE
Port Firefox 88.0.1 style to Firefox 89+?

### DISCUSSION THREAD

**Discussion by OmegaAOL** [Aris-t2/CustomCSSforFx#558] [Q&A]
Is there any way to simply use the same chrome folder with the new Proton UI?
Folder in question: https://github.com/OmegaAOL/Firefox3Theme/blob/main/chrome.zip

This makes FX Photon look like FX 3 with some changes. Can you suggest any alternative FX3 themes for FX89 and above?

**Comment by Aris-t2** [comment]
Please do not start new discussions. Use the one already available: https://github.com/Aris-t2/CustomCSSforFx/discussions/454

You can always combine CSS code from different sources by using it or linking to it inside `userChrome.css` or this projects `my_userChrome.css`.
If a "theme" needs to be updated for browsers current release, the author should be contacted.
